backend/app/services/rag/retrieval/__init__.py

```
@@ -2,18 +2,11 @@
 from app.services.rag.retrieval.filters import (
     build_elasticsearch_filters,
     parse_metadata_filters,
 )
-from app.services.rag.retrieval.retriever import DocumentRetriever
 
-__all__ = [
-    "DocumentRetriever",
-    "parse_metadata_filters",
-    "build_elasticsearch_filters",
-]
+__all__ = ["parse_metadata_filters", "build_elasticsearch_filters"]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)